### ISSUE TITLE
Laravel Mix build issues

### ISSUE THREAD

**Issue by UTCGilligan** [UTCWeb/utc-tailwind-genesis-theme#2]
## Proposed solution:
Please update the [package.json](https://github.com/UTCWeb/utc-tailwind-genesis-theme/blob/main/package.json) build scripts for Laravel Mix 6; hot and prod contain flags that are incompatible with Mix 6:
- https://stackoverflow.com/questions/65976497/npm-run-production-error-unknown-option-no-progress?answertab=votes#tab-top

## Research:
I'm having trouble running the web pack script (npm run prod) on the hosted environment, twebvmin11.is.utc.edu. I have Bridget's identical versions of npm (6) and node (14) running on that environment, but it looks like the version constraint for Laravel Mix in package.json may be too open, or the scripts need to be changed to use the newer Mix 6 directives, because it balks at --no-progress.
```
[twebblog@twebvmin11.is.utc.edu]: > cross-env NODE_ENV=production node_modules/webpack/bin/webpack.js --no-progress --config=node_modules/laravel-mix/setup/webpack.config.js
[twebblog@twebvmin11.is.utc.edu]:  [webpack-cli] Error: Unknown option '--no-progress'
``` 
- https://stackoverflow.com/questions/65976497/npm-run-production-error-unknown-option-no-progress?answertab=votes#tab-top
- https://github.com/UTCWeb/utcblogsmultisite/issues/13

**Comment by UTCGilligan** [comment]
[2022-01-24T23_00_01_146Z-debug.log](https://github.com/UTCWeb/utc-tailwind-genesis-theme/files/7933664/2022-01-24T23_00_01_146Z-debug.log)

**Comment by UTCGilligan** [comment]
Needs `npm rebuild` prior to running `npm run prod`

**Comment by UTCGilligan** [comment]
Deprecated flags still error out on hot and prod:
`[webpack-cli] Error: Unknown option '--inline'`
`[webpack-cli] Error: Unknown option '--no-progress'`

web pack versions are also identical hosted vs. lando/local

```
[twebblog@twebvmin11 utc-tailwind-genesis-theme]$ ./node_modules/.bin/webpack --version
webpack 5.48.0
webpack-cli 4.7.2
webpack-dev-server 4.0.0-beta.3
```
```
❯ ./node_modules/.bin/webpack --version
webpack 5.48.0
webpack-cli 4.7.2
webpack-dev-server 4.0.0-beta.3

```

prod errors out on local as well:
```
❯ npm run prod

> utc@1.0.0 prod
> npm run production


> utc@1.0.0 production
> cross-env NODE_ENV=production node_modules/webpack/bin/webpack.js --no-progress --config=node_modules/laravel-mix/setup/webpack.config.js

[webpack-cli] Error: Unknown option '--no-progress'
[webpack-cli] Run 'webpack --help' to see available commands and options

```

**Comment by UTCGilligan** [comment]
Closed by #4
